src/TagSelector.xml

```diff
@@ -23,6 +23,10 @@
                 <caption>Create tag microflow</caption>
                 <description>Microflow to be executed a new tag is added.</description>
             </property>
+            <property key="toLowercase" type="boolean" defaultValue="true">
```

Comment:
Name can be improved. Maybe `lowercaseInput` or similar.

Reply:
not relevant anymore property is removed